manage.py

```diff
@@ -22,14 +22,14 @@ def coverage_context():
     cov.save()
     print(f'Coverage is {cov.report()}%')
```

Comment:
It just occurred to me that this `cov.report()` call happens here for the first time and then, once again in the coverage level check. So you can fix the problem of double terminal output by calling once and reusing that variable in two places.

Reply:
Exactly, thank you

Reply:
Since you're also baking this change into the same PR, try to update the PR title and description to be more accurate too.

Reply:
Marking as resolved.